src/app/modules/core/advertisement/components/card/card-top/favourite/favourite.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="flex-container">
+  <button mat-icon-button color="warn" (click)="toggleSelected()">
+
+    <mat-icon aria-label="Example icon-button with a heart icon">
```

Comment:
questo aria-label lo toglierei

Reply:
In realtà andrebbe lasciato l'attributo `aria-label` visto che non c'è una label esterna. Andrebbe modificato il testo spiegando effettivamente cos'è quel bottone. Grazie